parser_library/src/debugging/debugger.cpp

```diff
@@ -387,11 +387,11 @@ class debugger::impl final : public processing::statement_analyzer
 
         for (const auto& [_, value_type] : proc_stack_[frame_id].scope.variables)
         {
-            const auto& [value, global] = value_type;
+            const auto& [ref, _data, global] = value_type;
```

Comment:
Consider using `__` instead
```suggestion
            const auto& [ref, __, global] = value_type;
```

Reply:
I had that and then decided to change it back :)